docs/en/observability/monitor-java-app.asciidoc

```diff
@@ -1469,7 +1469,7 @@ Results show that only twenty requests were sent, which makes sense
 given the processing time.
 +
 Now let’s build a visualization using
-https://www.elastic.co/guide/en/kibana/current/lens.html[Lens], a
+{kibana-ref}/dashboard.html#lens[Lens], a
 tool in {kib} allowing create visualizations easily.
```

Comment:
Not sure what the best fix is here, but the sentence doesn't make sense as is.

```suggestion
visualization tool in {kib}.
```

Reply:
@bmorelli25  How about - Now let's build a visualization using Lens in Kibana?

Reply:
Yup, that's better